docs/tutorials/Manual-Signature-Verification/Readme.md

```diff
@@ -0,0 +1,268 @@
+
+## Introduction
+
+When implementing [mobile banking authentication and authorization](https://github.com/wultra/wultra-docs/blob/develop/docs/tutorials/Authentication-in-Mobile-Apps/Readme.md), you need to implement at least two core processes:
+
+- [Activation](https://github.com/wultra/wultra-docs/blob/develop/docs/tutorials/Authentication-in-Mobile-Apps/Readme.md#activation) - mobile device enrollment
+- [Transaction signing](https://github.com/wultra/wultra-docs/blob/develop/docs/tutorials/Authentication-in-Mobile-Apps/Readme.md#transaction-signing) - for example, login or payment approval
+
+The **activation** process can be entirely externalized into a standalone [Enrollment Server](https://github.com/wultra/wultra-docs/blob/develop/docs/tutorials/Authentication-in-Mobile-Apps/Server-Side-Tutorial.md#deploying-the-enrollment-server) application. Enrollment server can take over the activation process and it can be deployed fully independently from your existing systems.
+
+The **transaction signing** process is much more tightly coupled with your protected API resources. As a result, you usually need to integrate the PowerAuth signature verification logic into your existing systems that publish those protected resources. This is a trivial task if you use Spring framework thanks to our magical `@PowerAuth` annotation, as we illustrate in our [tutorial on mobile authentication and transaction signing](https://github.com/wultra/wultra-docs/blob/develop/docs/tutorials/Authentication-in-Mobile-Apps/Server-Side-Tutorial.md#preparing-protected-api-resources).
+
+However, not all systems use Spring, or even Java. What if you use .NET, Java, Ruby, Python, or any other server-side technology? Do not worry - adding the support for manual ad-hoc signature verification is not difficult.
+
+
+## Mobile App Perspective
+
+To understand the server-side signature verification better, it is helpful to understand how the signatures are computed on the mobile device, using our iOS or Android SDK.
+
+When computing a signature, the mobile SDK requires the following input values:
+
+- **HTTP method**, usually `POST`.
+    - For signed requests, we always recommend using the `POST` method.
+- **Resource ID**, this a constant that client and server needs to agree on beforehand.
+    - The value of this constant is usually (by convention) set to the relevant part of the URI that gets called. For example, if the URI is `https://api.example.com/app-context/1.0/login`, the resource ID would be set to `/login` value.
+    - _Note: We use URI ID instead of the full URL since it could be difficult to deduce the correct path on the server side. Systems may run in different clusters and in different URI contexts (for example, the server could see `https://internal-machine/domain1/app-context/1.0-rel202005/login` instead of the above-mentioned address)._
```

Comment:
Might be worth mentioning that the URI ID does not have to be a substring of the full URI at all (in the example there is `login` shared between the URL and URI ID. I know it's not a good practice to use completely different names, but it can help the understanding.

Reply:
I improved the wording a bit...